modules/test/playwright/tests/frontend-data-set-views-web/helpers/DataSetManagerApiHelpers.ts

```diff
@@ -0,0 +1,74 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {ApiHelpers} from '../../../helpers/ApiHelpers';
+import {DEFAULT_LABEL} from '../utils/constants';
+
+const DEFAULT_DATA_SET_ERC = 'sampleDataSetERC';
+export class DataSetManagerApiHelpers extends ApiHelpers {
```

Comment:
I guess it's because `ApiHelpers` will be deleted if the headless PR gets merged one day so no one is thinking in refactoring or changing that... :shrug: